perftestnotebook.py

```diff
@@ -102,14 +105,15 @@ def parse_file_grouping(self, file_grouping):
 
         return files
 
-    def process(self):
+    def process(self, no_iodide):
```

Comment:
Sorry, one last thing to change is make the no_iodide an optional argument `no_iodide=True`.